packages/strapi/lib/Strapi.js

```diff
@@ -414,35 +416,48 @@ class Strapi {
     return reload;
   }
 
-  async runBootstrapFunctions() {
-    const execBootstrap = async fn => {
+  async runLifecyclesFunctions(functionName) {
+    const execFunction = async fn => {
       if (!fn) return;
 
       return fn();
     };
 
-    // plugins bootstrap
+    const configPath = `functions.${functionName}`;
+
+    // plugins register
```

Comment:
Here we have `plugins register`, `plugin bootstraps` and `pluginFunc`. Shouldn't all three be named `plugins functions`, `pluginFunctions`, and `pluginFunction` instead?